### PR TITLE
[WebRTC] Add new APIs for VideoFrameRate and bundle policy

### PR DESCRIPTION
**Pull request by hsgwon**
### Description of Change ###

Add VideoFrameRate property
Add BundlePolicy property and related enum

### API Changes ###

 - TCSACR-491

### CODE REVIEW

File: src/Tizen.Multimedia.Remoting/WebRTC/MediaSource.cs [hunk 1]
```diff
@@ -213,6 +213,70 @@ public Size VideoResolution
             }
         }
 
+        /// <summary>
+        /// Gets or sets the video frame rate of the current media source.
+        /// </summary>
+        /// <remarks>
+        /// This API is only supported in video media source, especially <a cref="MediaCameraSource"/>,
+        /// <a cref="MediaScreenSource"/>, <a cref="MediaTestSource(MediaType)"/>.<br/>
```
Comment (sangchul1011) [inline src/Tizen.Multimedia.Remoting/WebRTC/MediaSource.cs:221]: suggestion : MediaTestSource(MediaType) -> (MediaType) is it intended?
I think we could remove this.

File: src/Tizen.Multimedia.Remoting/WebRTC/MediaSource.cs [hunk 2]
```diff
@@ -213,6 +213,70 @@ public Size VideoResolution
...
+        /// </remarks>
+        /// <value>A value that specifies the video frame rate.</value>
+        /// <exception cref="ArgumentException">VideoFrameRate is less than zero.</exception>
```
Comment (sangchul1011) [inline src/Tizen.Multimedia.Remoting/WebRTC/MediaSource.cs:224]: suggestion: I think that 'less than or equal to zero' is right.

File: src/Tizen.Multimedia.Remoting/WebRTC/MediaSource.cs (hunk 1 above)
Comment (hsgwon) [reply src/Tizen.Multimedia.Remoting/WebRTC/MediaSource.cs]: Done

File: src/Tizen.Multimedia.Remoting/WebRTC/MediaSource.cs (hunk 2 above)
Comment (hsgwon) [reply src/Tizen.Multimedia.Remoting/WebRTC/MediaSource.cs]: Done

File: src/Tizen.Multimedia.Remoting/WebRTC/WebRTC.Properties.cs [hunk 3]
```diff
@@ -199,5 +199,32 @@ public IceTransportPolicy IceTransportPolicy
                     ThrowIfFailed("Failed to set ICE transport policy");
             }
         }
+
+        /// <summary>
+        /// Gets or sets the bundle policy.<br/>
+        /// The default bundle policy is <a cref="WebRTCBundlePolicy.MaxBundle"/>.
```
Comment (wiertel) [inline src/Tizen.Multimedia.Remoting/WebRTC/WebRTC.Properties.cs:205]:
```suggestion
        /// The default bundle policy is <see cref="WebRTCBundlePolicy.MaxBundle"/>.
```

File: src/Tizen.Multimedia.Remoting/WebRTC/MediaSource.cs [hunk 4]
```diff
@@ -213,6 +213,70 @@ public Size VideoResolution
             }
         }
 
+        /// <summary>
+        /// Gets or sets the video frame rate of the current media source.
+        /// </summary>
+        /// <remarks>
+        /// This API is only supported in video media source, especially <a cref="MediaCameraSource"/>,
+        /// <a cref="MediaScreenSource"/>, <a cref="MediaTestSource"/>.<br/>
```
Comment (wiertel) [inline src/Tizen.Multimedia.Remoting/WebRTC/MediaSource.cs:221]:
```suggestion
        /// This API is only supported in video media source, especially <see cref="MediaCameraSource"/>,
        /// <see cref="MediaScreenSource"/>, <see cref="MediaTestSource"/>.
```

File: src/Tizen.Multimedia.Remoting/WebRTC/WebRTC.Properties.cs (hunk 3 above)
Comment (hsgwon) [reply src/Tizen.Multimedia.Remoting/WebRTC/WebRTC.Properties.cs:205]: Done

File: src/Tizen.Multimedia.Remoting/WebRTC/MediaSource.cs (hunk 4 above)
Comment (hsgwon) [reply src/Tizen.Multimedia.Remoting/WebRTC/MediaSource.cs:221]: Done

**Review by aferin — APPROVED** [review]

**Pull request merged**